superbench/benchmarks/docker_benchmarks/docker_base.py

```diff
@@ -20,8 +21,15 @@ def __init__(self, name, parameters=''):
         """
         super().__init__(name, parameters)
         self._benchmark_type = BenchmarkType.DOCKER
+
         # Command lines to launch the docker image and run the benchmarks inside docker.
-        self.__commands = list()
+        self._commands = list()
+
+        # Image url of the current docker-benchmark.
+        self._image = None
+
+        # Container name of the current docker-benchmark.
+        self._container = None
```

Comment:
How to set image and container?

Reply:
In derived class, just like `self._bin_name` in `MicroBenchmarkWithInvoke`, which is forced to set.